contracts/examples/digital-cash/src/digital_cash.rs

```diff
@@ -30,6 +30,11 @@ pub trait DigitalCash {
     fn fund(&self, address: ManagedAddress, valability: u64) {
         let deposit_mapper = self.deposit(&address);
         require!(!deposit_mapper.is_empty(), FEES_NOT_COVERED_ERR_MSG);
+        let depositor = deposit_mapper.get().depositor_address;
```

Comment:
we have to simplify - take fees in the fund function, should not need to make 2 txs.

Reply:
we cannot make eGLD + ESDT in 1 transaction

Reply:
but you can take, wEGLD in case of ESDT.